pinot-core/src/test/java/org/apache/pinot/core/common/datatable/DataTableSerDeTest.java

```diff
@@ -164,7 +170,7 @@ public void testAllDataTypes()
   }
 
   @Test
-  public void testV3V4Compatibility()
+  public void testV2V3Compatibility()
```

Comment:
Can we revert the reordering of V2V3 and V3V4 compatibility test? It is really hard to track the changes here